### ISSUE TITLE
Legacy intrinsics

### ISSUE THREAD

**Issue by ArchangeGabriel** [intel/intel-graphics-compiler#277]
While building 1.0.12812.4 with LLVM15 and vc-intrinsics 0.11.0 I’m facing this:
```
error: <llvm.genx.rdregioni.i64.v1i64.i16> intrinsic is unsupported by target platform <SKL>. Please, enable legacy intrinsics translation
error: <llvm.genx.rdregioni.i64.v1i64.i16> intrinsic is unsupported by target platform <BDW>. Please, enable legacy intrinsics translation
error: <llvm.genx.rdregioni.i64.v1i64.i16> intrinsic is unsupported by target platform <GLK>. Please, enable legacy intrinsics translation
```
I’ve tried to look how to “enable legacy intrinsics translation” but found nothing relevant.

**Comment by ArchangeGabriel** [comment]
OK, the issue is likely https://github.com/intel/vc-intrinsics/commit/bda6db9b23d695998faac74ace4094adb285ca28. I guess igc will have to adapt to this, in the meantime I’ll just revert locally.

**Comment by frantisekz** [comment]
Later igc tags work just fine with the latest vc-intrinsics (at least the igc-1.0.12812.21).

**Comment by ArchangeGabriel** [comment]
OK, so I’ll consider this already fixed in master but #273 blocking its use… I still don’t know what version I should package exactly, so I followed you and went with 1.0.12812.4…
